site/_layouts/default.html

```diff
@@ -5,7 +5,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta name="description" content="{{site.description}}">
     <meta name="author" content="Marcus Ilgner">
-    <meta http-equiv="Content-Security-Policy" content="script-src 'self';img-src 'self';style-src 'unsafe-inline' 'self';">
+    <meta http-equiv="Content-Security-Policy" content="script-src 'self';img-src 'self' img.shields.io;style-src 'unsafe-inline' 'self';">
```

Comment:
Should be a header ideally... :thinking:

Reply:
I was sure to have added the shields img host!?

Reply:
I'll think about it... :thinking: